app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  after_create :set_default_role
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
```

Comment:
- [ ] Let's not use commented code in our project.

Reply:
I'm a code reviewer too and excuse me but the rules say we can't do the next requested changes after the student completed the requested changes, and next should be an optional thing, so these requested changes are invalid, therefore, it should be project approved, due that I have completed the first reviewer requested changes already.
Is Forbidden and in the code reviewers rules not to ask a next requested changes in second review.

Reply:
"SpaYco:rocket:  6:25 AM
hey, if the previous reviewer didn't mention an issue that you spotted, it would become optional"

Reply:
I'm a code reviewer too, rules say we can't request "requested changes to students" after the student already completed the previous reviewer requested changes, especially in the second review, the second review should be optional changes and project approved.  Is Forbidden and in the code reviewers rules not to ask for requested changes in the second code review after the student completed all requested changes of the previous reviewer. Therefore this should be project approved.

Reply:
"SpaYco🚀 6:25 AM
hey, if the previous reviewer didn't mention an issue that you spotted, it would become optional"

Reply:
thanks for pointing it out.

Reply:
Thanks for pointing it out, I think that should be an optional change since isn't in the project requirements, therefore it should've been project approved.  I will take into consideration the comments for the next milestone.